seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-2/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcOceanBaseMysqlIT.java

```diff
@@ -280,6 +278,7 @@ String getFullTableName(String tableName) {
     @Override
     GenericContainer<?> initContainer() {
         return new GenericContainer<>(IMAGE)
+                .withEnv("MODE", "slim")
```

Comment:
Any different before and after this change? What's default `MODE` value?

Reply:
The mode `slim` is introduced since 4.2.0.0 https://hub.docker.com/r/oceanbase/oceanbase-ce. For now only the OceanBase container on this mode can be used on GitHub workflow.

Reply:
Thanks for explain it @whhe . I got it.